mining.md

```diff
@@ -113,59 +113,81 @@ In order to validate a block coming in from the network at round `N` was well mi
 
 ```go
 func VerifyBlock(blk Block) {
```

Comment:
Is there a `BuildBlock` method somehwere?

Reply:
I think we don't have pseudocode for it right now, just the prose description down below. It might be good to have pseudocode for that tho